peps/pep-0738.rst

```diff
@@ -269,11 +284,24 @@ by ``os.uname()``, with the exception of:
 In addition, a ``platform.android_ver()`` method will be added, which returns a
 namedtuple containing the following:
 
-* ``release`` - Android version, as a string (e.g. ``"14"``)
-* ``api_level`` - Android API level, as an integer (e.g. ``34``)
+* ``release`` - Android version of the device, as a string (e.g. ``"14"``)
+* ``api_level`` - :ref:`API level <738-os-versions>` of the device, as an
+  integer (e.g. ``34``)
```

Comment:
What's the future of `sys.getandroidapilevel()`? If the API Level is available here in the platform module, should we deprecate the `sys` mechanism? Personally, I've always found the `getandroidapilevel()` method to be an odd wart, so I'd be in favor of deprecating it; but I guess it doesn't *hurt* to keep it around.

Reply:
I agree, but it's been the de-facto mechanism to detect Android for so long now, that I think it would be too disruptive to remove. Virtually every package that's ever added Android-specific code is using it, and we wouldn't want to start off the era of official CPython Android support by breaking them all, even after a 2 year deprecation period.